### ISSUE TITLE
Unhandled error starting server: 'dict' object is not callable

### ISSUE THREAD

**Issue by zi-dan** [jupyterhub/dockerspawner#291]
Hi Guys,

I just upgrade to latest release (0.11.0) but spawning a swarm container stopped with exception below. I have tested with JupyterHub version 0.92, 0.94 and latest 1.0.dev (docker latest build jupyterhub/jupyterhub:1.0.dev)

```
[E 2019-03-04 14:01:06.580 JupyterHub pages:148] Failed to spawn single-user server with form
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/handlers/pages.py", line 146, in post
        await self.spawn_single_user(user, options=options)
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/handlers/base.py", line 705, in spawn_single_user
        timedelta(seconds=self.slow_spawn_timeout), finish_spawn_future
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/handlers/base.py", line 626, in finish_user_spawn
        await spawn_future
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/user.py", line 489, in spawn
        raise e
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/user.py", line 409, in spawn
        url = await gen.with_timeout(timedelta(seconds=spawner.start_timeout), f)
      File "/opt/conda/lib/python3.6/site-packages/dockerspawner/dockerspawner.py", line 984, in start
        obj = yield self.create_object()
      File "/opt/conda/lib/python3.6/site-packages/dockerspawner/swarmspawner.py", line 179, in create_object
        args=(yield self.get_command()),
      File "/opt/conda/lib/python3.6/site-packages/dockerspawner/dockerspawner.py", line 812, in get_command
        return cmd + self.get_args()
      File "/opt/conda/lib/python3.6/site-packages/dockerspawner/dockerspawner.py", line 726, in get_args
        args = super().get_args()
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/spawner.py", line 700, in get_args
        notebook_dir = self.format_string(self.notebook_dir)
      File "/opt/conda/lib/python3.6/site-packages/jupyterhub/spawner.py", line 681, in format_string
        return s.format(**self.template_namespace())
    TypeError: 'dict' object is not callable
```

Any idea for resolve this?

**Comment by pinsleepe** [comment]
I have the same issue...

**Comment by eaton-lab** [comment]
I was able to avoid this error by removing `c.DockerSpawner.notebook_dir = '...'` from my config.

**Comment by MAliNaqvi** [comment]
@eaton-lab since this notebook_dir could also be in a `c.DockerSpawner.pre_spawn_hook`, there is a check for notebook_dir there. So I don't think removing the notebook_dir is a good solution.

**Comment by Victorsoukhov** [comment]
I have the same issue

**Comment by vlizanae** [comment]
this is the same as #290

**Comment by minrk** [comment]
Fixed by #296, released as 0.11.1
